src/styles.css

```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import "https://fonts.googleapis.com/icon?family=Material+Icons";
```

Comment:
You can add `<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">` in index.html file instead of import here.